lib/unifex/code_generator.ex

```diff
@@ -45,9 +49,8 @@ defmodule Unifex.CodeGenerator do
   defp get_generator_module_name(interface) do
     module_name =
       case interface do
-        :nif -> :NIF
-        :cnode -> :CNode
-        other -> other
```

Comment:
This line was for user defined generators as well as for these specified in `*.spec.exs` file. Removing it we will probably support only `:nif` and `:cnode` generators specified in `bundlex.exs` :roll_eyes: